components/organisms/ProfileTabBody.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  Button,
+  CalendarSVG,
+  CogSVG,
+  EthTransparentSVG,
+  HeartSVG,
+  Modal,
+  Skeleton,
+} from "@ensdomains/thorin";
+import { ProfileRecordItem } from "../02-molecules";
+import { DatabaseIcon } from "../01-atoms";
+import { formatDate, formatHexAddress } from "@/lib/utils/formats";
+import { CoinInfo, DomainData } from "@/lib/domain-page";
+import { useAccount, useEnsName } from "wagmi";
+import { useState } from "react";
+import { EditResolverModalContent } from "./EditResolverModalContent";
+import { useRouter } from "next/router";
+import { excludeKeys } from "@/pages/domains/[name]";
+
+export interface ProfileTabProps {
+  domainData: DomainData | null;
+}
+
+export const ProfileTabBody = ({ domainData }: ProfileTabProps) => {
+  const [editResolverModalOpen, setEditResolverModalOpen] = useState(false);
+  const { address } = useAccount();
+
+  const router = useRouter();
+  const { name } = router.query; // Dynamic route parameter
```

Comment:
Can you put Dynamic route docs here as well?

Reply:
@FrancoAguzzi  Isn't that a bit excessive? It's like including documentation from React or Next.js for every built-in function we use